tICA/README.md

```diff
@@ -0,0 +1,19 @@
+# Temporal ICA
+
+This pipeline uses ICA on a large amount of fMRI data (typically a large group of subjects),
+in order to separate global nuisance effects (such as due to respiration) from neural signals that
+have a nonzero spatial mean.
+
+The processing modes also allow for applying temporal ICA cleanup to datasets that are not
+large enough to reliably produce a new temporal ICA decomposition.  Briefly, the processing
+modes are:
+
+- NEW - do a full pipeline run from scratch, starting with a fresh MIGP, intended for large datasets
+
+- REUSE_SICA_ONLY - use a previous MIGP result, but estimate the temporal ICA from scratch
+
+- INITIALIZE_TICA - use a previous MIGP result, and estimate temporal ICA starting from a previous
+tICA mixing matrix, but allow it to change to fit the new data
```

Comment:
Can we elaborate on what it means to "allow it to change to fit"?  Presumably this mode starts with an existing classification, but then somehow "tweaks" it in some fashion to make it a bit better customized to the data at hand?

Reply:
This is pretty close to the weeds, it is sort of an implementation detail of how ICA works.  The two modes that don't say "intended for" may be rather niche cases, and may deserve to be marked as such.

Reply:
The actual implementation is ICASSO, which uses FastICA (which uses their own fixed-point algorithm), with random starting points by default.  I believe the `INITIALIZE_TICA` mode skips having ICASSO run with multiple starting points (for stability when using random starting points), and instead runs only once with the matrix from the prior dataset as the starting point.  @glasserm ?

Reply:
Initialize still does the ICASSO, just with a specific initialization matrix.

Reply:
Right, but it doesn't do the multiple rounds it does when using random initialization, yes?

Reply:
No it still does.  All it skips is the first totally random initializations.

Reply:
By "multiple rounds" I meant the 100(?) runs on random seeds (shouldn't need 100 runs on the same starting point) - our additional iterations outside that are an additional complication which I never really tried to understand.  So, really, a tidy explanation has to be hand-wavy, because the details are messy.

Reply:
An initialization matrix (which is also what we have after 100 random runs) will still likely not be perfectly orthogonal.  The point of all the ICASSO stuff is to generate reproducible decompositions.

Reply:
The details may be messy, but I think we should capture some flavor of this in the `README`

Reply:
What type of user needs more of the implementation details than "allow it to change to fit the new data" for this particular tangent?  How is it important for the purpose of getting the pipeline to run?

Recommendations would be fine (@glasserm has a better idea than me what to recommend), but I don't think this kind of gritty detail belongs in "how to use it" - if you want to understand the details of how it works, read the paper, then the code/comments.

Reply:
I think we should just say what the options do and not make explicit recommendations here.

Reply:
Speaking of the paper, @glasserm is https://www.sciencedirect.com/science/article/abs/pii/S1053811918303963 the reference we want to put in the readme for tICA?

Reply:
Yes

Reply:
As a user, I'd like to know what situations should prompt me to consider using a given mode, without having to read the paper, or the code/comments.

Reply:
We haven't written a paper describing when each mode should be used.  That will be in Alex's paper.  We haven't fully defined this yet.